apps/examples/smartfs/fs_performance/fs_performance_main.c

```diff
@@ -0,0 +1,461 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <tinyara/timer.h>
+
+/****************************************************************************
+ * Pre Processor Definitions
+ ****************************************************************************/
+
+#define ROOT				"/mnt/"
+#define TEST_FILE			ROOT "test_file"
+#define TEST_FLAG_FILE_CREATION		1
+#define TEST_FLAG_OVERWRITE		2
+#define TEST_FLAG_READ			3
+#define TEST_FLAG_GC_EFFECT		4
+#define TEST_FLAG_APPEND		5
+#define TEST_FILE_NAME_LEN_MAX		15
+#define ITR_OVE				5
+#define ITR_APP				5
+#define ITR_READ			100
+#define ITR_GC				10
+#define N_MULTIPLE_FILES		2
+#define TIMER_DEVNAME			"/dev/timer%d"
+#define MAX_RD_BUF_SIZE			32768
+#define MAX_BUF_SIZE			65536
+#define MAX_FILE_SIZE			65536
+
+/****************************************************************************
+ * Public Variables
+ ****************************************************************************/
+
+int f_sz;
+int b_sz;
+char file[N_MULTIPLE_FILES][TEST_FILE_NAME_LEN_MAX];
+char rw_buf[MAX_BUF_SIZE];
+char read_buf[MAX_RD_BUF_SIZE];
+char timer_path[_POSIX_PATH_MAX];
```

Comment:
it's suggestion. how about add 'g_' prefix to those global variables to make easy to read.

Reply:
Thankyou for your suggestion. Updated